### ISSUE TITLE
Make `--emit` and `--option` consitent in the config

### ISSUE THREAD

**Issue by timotheeguerin** [microsoft/cadl#1274]
Designed in  #1213
Replace `emitters` in `cadl-project` with 2 new settings being consitent with what you can do in the cli

```yaml
# Select the emitter
emit: 
 - /path/to/other-emitter

# Sepecfy the options for emitter: Doesn't enable emitters
options:
  @cadl-lang/openapi3:
    output-dir: abc
  other-emitter:
    option1: true
```

**Comment by markcowl** [comment]
look up issue for error when emitter not found

est: 5

**Comment by markcowl** [comment]
Fix with #1211
